src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseDataFileServiceTest.cs

```diff
@@ -2866,9 +2984,30 @@ await using (var statisticsDbContext = InMemoryStatisticsDbContext(statisticsDbC
                 Assert.Equal(IStatus.QUEUED, result.Right.Status);
             }
 
-            await using (var context = InMemoryApplicationDbContext(contentDbContextId))
+            await using (var statisticsDbContext = InMemoryStatisticsDbContext(statisticsDbContextId))
+            {
+                var subjects = statisticsDbContext.Subject.ToList();
+                Assert.Equal(2, subjects.Count);
+                Assert.NotNull(subjects.First(s => s.Id == originalSubject.Id));
```

Comment:
I would have used `find` here rather than `first`. First is going to throw an exception before reaching the NotNull check if it's not existing. Same in the previous test above, line 2593.